index.js

```diff
@@ -10,12 +10,11 @@ var {
 function JMON(data) {
 
   this.initialData = data;
-  this.data = _.cloneDeep(data);
+  this.data = _.cloneDeep(this.initialData);
 
-  // Containers
-  this.createdList = [];
-  this.updatedList = [];
-  this.deleteList = [];
+  this.isCreated = false;
```

Comment:
@bibekluitel I think this is wrong.
These values we will be adding at prototype, right?